packages/authentication-oauth/src/utils.ts

```diff
@@ -1,38 +1,24 @@
-import querystring from 'querystring';
+import { RequestHandler } from 'express';
+import session from 'express-session';
 import { Application } from '@feathersjs/feathers';
-import { AuthenticationService, AuthenticationResult } from '@feathersjs/authentication';
 
 export interface OauthSetupSettings {
-  path?: string;
-  authService?: string;
-  linkStrategy?: string;
-  getRedirect? (service: AuthenticationService, data: AuthenticationResult|Error): Promise<string>;
+  path: string;
+  authService: string;
+  linkStrategy: string;
+  expressSession: RequestHandler;
```

Comment:
That doesn't look like the right type?

Reply:
It's `import { RequestHandler } from 'express';` right? It at least doesn't complain when initializing the default options in https://github.com/feathersjs/feathers/pull/1335/files/40ebbddeae3995e840c059146a6e3d3597087107#diff-b3c31ade33c8365501f61c7361f117b2R17

Reply:
Ah, I guess this makes sense if expressSession is like a middleware (RequestHandler).